src/js/tabs/osd.js

```diff
@@ -1806,6 +1885,19 @@ OSD.chooseFields = function() {
                                                                 F.AUX_VALUE,
                                                                 F.READY_MODE,
                                                             ]);
+                                                            if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_46)) {
```

Comment:
```suggestion
                                                            if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_45)) {
```

Reply:
But the current FW is 1.46 MSP which is there the F.SYS stuff got added.

Reply:
We should target this for 1.45